crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1583,139 +1593,144 @@ impl TryFrom<StoredTransaction> for TransactionBlock {
     }
 }
 
-impl TryFrom<StoredEpochInfo> for Epoch {
```

Comment:
This TryFrom is replaced with a free function as we need to pass validators_apy data to create the active validators.